addons/flux-cronjob-deployment/apps/igr/cronjobs-titancli/<cronjob_name>.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: @{{ cronjob_name }}
+spec:
+  schedule: "@{{ cronjob_schedule }}"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: @{{ cronjob_name }}
+          labels:
+            app: @{{ cronjob_name }}
+        spec:
+          containers:
+            - name: titancli
+              image: tamediapmd-cloud.jfrog.io/titan-cli:igr-8c3166a-1676450548 # {"$imagepolicy": "flux-system:igr-titan-cli"}
+              args:
+                - @{{ container_arg }}
```

Comment:
@Davidoutz With an extra effort, we could support another use case (multi-args) that is already present in Titan (e.g. [subscriptionsyncqueue](https://github.com/tx-group-adm/titan-flux-infra/blob/main/apps/igr/cronjobs-titancli/titancli-subscriptionsyncqueue-high-load.yaml#L17))

Using Jinja lists like done in the [template-generic-app](https://github.com/tx-pts-dai/template-generic-app/blob/main/.github/workflows/application.yaml#L32). I think it's worth it.